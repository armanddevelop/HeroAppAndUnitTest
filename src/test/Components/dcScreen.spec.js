import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { DcScreen } from "../../Components/DC/DcScreen";

describe("test to DcScreen component", () => {
  const container = shallow(<DcScreen />);
  it("Snapshot to DcScreen component", () => {
    expect(toJson(container)).toMatchSnapshot();
  });
});
