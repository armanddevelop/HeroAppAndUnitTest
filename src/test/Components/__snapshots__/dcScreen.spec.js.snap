// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to DcScreen component Snapshot to DcScreen component 1`] = `
<Fragment>
  <h1>
    DC Screen
  </h1>
  <hr />
  <HeroList
    publisher="DC Comics"
  />
</Fragment>
`;
